packages/components/button/tests/button-group.test.tsx

```diff
@@ -69,3 +69,55 @@ test("Should flush outline button", () => {
     marginInlineEnd: "",
   })
 })
+
+test("Should flush vertical button", () => {
+  const { getByText } = render(
+    <ButtonGroup isAttached isVertical>
+      <Button>Button 1</Button>
+      <Button>Button 2</Button>
+      <Button>Button 3</Button>
+      <Button>Button 4</Button>
+    </ButtonGroup>,
+  )
+  expect(getByText(/Button 1/i)).toHaveStyle({
+    borderBottomLeftRadius: "0",
+    borderBottomRightRadius: "0",
+  })
+  expect(getByText(/Button 2/i)).toHaveStyle({
+    borderRadius: "0px",
+  })
+  expect(getByText(/Button 3/i)).toHaveStyle({
+    borderRadius: "0px",
+  })
+  expect(getByText(/Button 4/i)).toHaveStyle({
+    borderTopLeftRadius: "0",
+    borderTopRightRadius: "0",
+  })
+})
+
+test("Should flush vertical outline button", () => {
+  const { getByText } = render(
+    <ButtonGroup isAttached isVertical variant="outline">
+      <Button>Button 1</Button>
+      <Button>Button 2</Button>
+      <Button>Button 3</Button>
+      <Button>Button 4</Button>
+    </ButtonGroup>,
+  )
+  expect(getByText(/Button 1/i)).toHaveStyle({
+    marginInlineEnd: "",
+    // marginBottom: "-1px",
+  })
+  expect(getByText(/Button 2/i)).toHaveStyle({
+    marginInlineEnd: "",
+    // marginBottom: "-1px",
+  })
+  expect(getByText(/Button 3/i)).toHaveStyle({
+    marginInlineEnd: "",
+    // marginBottom: "-1px",
+  })
```

Comment:
I have tried a crazy amount of combinations to get `marginBottom: "-1px"` to pass and think I might benefit from a contributor's guidance on this. I can confirm in the story (also a committed file) that this is the correct CSS value present, but for the life of me cannot get this attribute to pass. I could not find a way to print out the styles clearly either, could someone give me a pointer on this property/tip to get this test working?